### DISCUSSION TITLE
Does Ink support text formatting, e.g. bold/italics?

### DISCUSSION THREAD

**Discussion by hola-estif** [inkle/ink#832] [Writing in ink] [answered]
Hi there,

Does Ink support text formatting like bold and italics? There are places where I want to emphasise certain words in the dialogue but I don't want to use caps. I tried using markdown but that doesn't seem to work. Couldn't find it in the documentation either - apologies if I've missed this guidance though. 

Side note: I started writing with Ink 6 months ago and I'm absolutely loving it. Thank you for creating such an awesome tool and making it open source. 

Many thanks in advance, 
Steve

**Comment by Selsynn** [comment, marked as answer]
Ink is mostly about the game logic and what text to change. How you display it is up to yourself, because it will be technology dependant. 
If you are doing a webgame with the inkjs framework. I do believe you can use HTML markup. 

For example 
```
<strong>Once</strong> <em>upon</em> <b>a</b> time...

 * There were two choices.
 * There were four lines of content.

- They lived happily ever after.
    -> END

```
Gives that result
![image](https://user-images.githubusercontent.com/7371599/227743578-6ec0fc8f-be80-4721-bbbe-55dd79c7d658.png)

So it seems the `b` for bold tag does not reflect as in Inky. 
but you can use `Em` for emphasis (italic) and `strong` for more bold font.

If you have whole paragraph with that style, I would advise to use the class system. (like it's explained in the Custom CSS classes here https://www.inklestudios.com/ink/web-tutorial/ )

Note: everything is about the HTML export of the story. Not any other export where you would have to search how to markup your text with the technology you use.

**Reply by hola-estif** [reply]
Thanks for the comprehensive answer! Exactly what I was looking for, thanks so much.
